Cargo.toml

```diff
@@ -192,6 +192,10 @@ test_util = { workspace = true }
 tokio = { version = "1.17", features = ["macros", "rt-multi-thread", "time"] }
 zipf = "6.1.0"
 
+[dependencies.rocksdb]
```

Comment:
Should use engine_rocks::raw instead.

Reply:
> Should use engine_rocks::raw instead.

^^^^^^^^^ no `IndexType` in `raw`.  Do I need to change rust-rocks lib?

Reply:
raw is a re-export of needed rust-rocksdb structs. You can add `IndexType` in raw.rs.